test-projects/konsist-path-tester/data/src/test/kotlin/com/lemonappdev/sample/LibClassSpec.kt

```diff
@@ -1,3 +1,3 @@
 package com.lemonappdev.sample
 
-class LibClassTest
```

Comment:
It would be better to keep the original class name as it is used to validate existing test selection logic.

Rename it back to `LibClassTest` and add new `LibClassSpec` used for these tests